paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerSheetViewState.kt

```diff
@@ -120,9 +121,9 @@ internal sealed class CustomerSheetViewState(
         isProcessing = isProcessing,
         isEditing = false,
         screen = if (isFirstPaymentMethod) {
-            PaymentSheetScreen.AddFirstPaymentMethod
+            PaymentSheetScreen.AddFirstPaymentMethod(interactor = UnsupportedAddPaymentMethodInteractor)
         } else {
-            PaymentSheetScreen.AddAnotherPaymentMethod
+            PaymentSheetScreen.AddAnotherPaymentMethod(interactor = UnsupportedAddPaymentMethodInteractor)
```

Comment:
I don't really like doing this. It should be really obvious if we ever depend on the interactor in a real way, so I think this is _fine_, but we could also:
- make the interactor nullable on these screens and set it to null here. Then we could just make the `Content` for the screens be empty if the interactor is null
- Add an enum to `PaymentSheetScreen` with a value for each screen. We could it use for `HeaderTextFactory`, `PaymentSheetTopBarStateFactory`, etc, which don't really depend on the whole screens and just want to know _which_ screen we're using.

Curious if you have an opinion! If I was to add the enum to PaymentSheetScreen, I'd probably prefer to do it in a follow up since it's otherwise unrelated to this PR